src/app/modules/shared-components/components/input-error/input-error.component.html

```diff
@@ -0,0 +1,5 @@
+<div *ngIf="control && control.errors" style="color: #a94442">
```

Comment:
Good idea to make a component for that. Why "input" if you want to reuse it not only for 'input' but also in other types of form elements??

Reply:
The style thing should be in the CSS file.

Reply:
This component can be used with a component that allow input, but I can see why it could be confusing, what do you think would be a better name ?
Also, the css has been moved.

Reply:
FormError ?